code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts

```diff
@@ -296,7 +296,9 @@ export default async (
             fullySpecified: false,
           },
         },
-        createBabelLoader(babelOptions, typescriptOptions),
+        builderOptions.useSWC
+          ? createSWCLoader({})
```

Comment:
So we don't want to enable the users to pass any swc-related options aren't there any?

Reply:
Not part of this PR yet, will need to happen, agreed.

Reply:
Actually I think we might not want to.. and instead let users configure it using the SWC config file only.
Ensuring the setting is the same, and we do the minimum amount to get it to work out of the box.